stats.lua

```diff
@@ -392,6 +402,54 @@ local function append_display_sync(s)
 end
 
 
+local function append_filters(s, prop, prefix)
+    local length = 0
+    local filters = {}
+
+    local speed = mp.get_property_osd("speed")
+    if speed ~= "1.00" then
```

Comment:
I mostly added it cause it "feels" like a filter to me. I'm open to remove it, or put it somewhere else in the info.
Also, at least for audio, it will insert the scaletempo filter. But as with any other internal filters inserted by mpv, there seems to be no af/vf property to inspect them and the args they have.
I did make a feature request for something like that: https://github.com/mpv-player/mpv/issues/4659